src/tests/run-study-tests/readme.md

```diff
@@ -215,7 +215,7 @@ The schema can be found at : **src/tests/run-study-tests/parse_studies/json_sche
 
 ```bash
 > cd src/tests/run-study-tests
-> python -m pytest -m json --solver-path=/path/to/the/Antares/solver/antares-x.y-solver.exe
+> python -m pytest -m json --antares-simu-path=/path/to/the/Antares/solver/antares-x.y-solver.exe
```

Comment:
antares-simu-path is misleading. I read it as "antares simulation path", not "antares simulator path"